### PR TITLE
Show script filename instead of thumbnail

### PR DESCRIPTION
**Pull request by volzhs**
script1
script2

### CODE REVIEW

File: editor/property_editor.cpp [hunk 1]
```diff
@@ -3642,8 +3646,12 @@ void PropertyEditor::update_tree() {
 						item->set_tooltip(1, res->get_name() + " (" + res->get_class() + ")");
 					}
 					if (!res->is_class("Texture")) {
-						//texture already previews via itself
-						EditorResourcePreview::get_singleton()->queue_edited_resource_preview(res, this, "_resource_preview_done", item->get_instance_ID());
+						if (res->is_class("Script")) {
+							item->set_text(1, res->get_path().get_file());
+						} else {
+							//texture already previews via itself
```
Comment (bojidar-bg) [inline editor/property_editor.cpp]: Leave this comment where it was, there is no point in moving it around.
Also, **might** look better as: (but I'm not sure if it is much of an improvement...)
```cpp
if (res->is_class("Texture")) {
  //texture already previews via itself
} else if (res->is_class("Script")) {
  item->set_text(1, res->get_path().get_file());
} else {
  EditorResourcePreview::get_singleton()->queue_edited_resource_preview(res, this, "_resource_preview_done", item->get_instance_ID());
}
```
Or:
```cpp
if (res->is_class("Script")) {
  item->set_text(1, res->get_path().get_file());
} else if (!res->is_class("Texture")) {
  //texture already previews via itself

  EditorResourcePreview::get_singleton()->queue_edited_resource_preview(res, this, "_resource_preview_done", item->get_instance_ID());
}
```

Comment (volzhs) [inline editor/property_editor.cpp]: @bojidar-bg 
There are several `item->set_text(1, ...)` on upper lines.
I was afraid to make any side effect (bug) with what you suggested `if else` code.
I'm not 100% sure it's safe.

Comment (bojidar-bg) [inline editor/property_editor.cpp]: I think the logic executed is the same, just organized in a different way. Can't prove it tho.

Comment (volzhs) [inline editor/property_editor.cpp]: @bojidar-bg 
I would take second one then.
I will push this commit again, and use it for a while.
let's see if it's ok. :smile:

**Pull request merged**